### PR TITLE
test: (e2e) added feed input e2e core tests

### PR DESCRIPTION
**Pull request by nikvalor**
#### Please provide a link to the associated issue.
feed-input: [#4920](https://app.zenhub.com/workspaces/fundamental-library-5f32933d64202b0012e3572e/issues/sap/fundamental-ngx/4920)
#### Please provide a brief summary of this pull request.
Added feed input e2e tests core

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by nikvalor** [comment]
Project: null Platform B
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform A 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

**Review by InnaAtanasova — COMMENTED** [review]
Please fix your PR title

**Comment by nikvalor** [comment]
Project: null Platform B
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform A 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

**Comment by InnaAtanasova** [comment]
@nikvalor please address the checkboxes in the PR Template. Mergeable will not pass without this.

**Comment by droshev** [comment]
@nikvalor We would need a doc explaining how to:
- address and fix failing tests
- regenerate new snapshots
- what needs to be done if we bring markup changes
- how we handle fork PRs
without a doc explaining all above we should not accept new tests

File: e2e/wdio/core/fixtures/testData/feed-input-tags.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export const activeInputButtonsFocusState = 'active-input-button-focus-state-' + getImageTagBrowserPlatform();
+export const activeInputButtonsActiveState = 'active-input-button-active-state-' + getImageTagBrowserPlatform();
+
+
+
+
+
+
+
+
```
Comment (Seamoo13) [inline e2e/wdio/core/fixtures/testData/feed-input-tags.ts:22]: please remove extra lines

File: e2e/wdio/core/tests/feed-input.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    placeholder_array, default_avatar, send_button_tooltip,
+    button, textArea
+} from '../fixtures/appData/feed-input-page-contents';
+
+describe('Verify Feed Input component', function() {
+    const {
+        feedInputTextArea, feedInputAvatar, feedInputButton, feedInputNoAvatar, disableInputTextArea,
+        disableInputButton, activeInputTextAreas
+    } = new FeedInputPo();
+    const feedInputPage = new FeedInputPo();
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/feed-input.e2e-spec.ts:48]: please create PO first
const feedInputPage = new FeedInputPO;
const {} = feedInputPage

to keep the same style in the tests

File: e2e/wdio/core/tests/feed-input.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(feedInputAvatar);
+    }, 1);
+
+    it('should have correct placeholder assigned', () => {
+        const feedInputLength = getElementArrayLength(feedInputTextArea);
+        for (let i = 0; i < feedInputLength; i++) {
+            expect(getAttributeByNameArr(feedInputTextArea, 'placeholder')).toEqual(placeholder_array);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/feed-input.e2e-spec.ts:62]: this only checks index 0, need to do like this:
expect(getAttributeByNAmeArr(feedInputTextArea, 'placeholder', i)).toEqual(placeholder_array[i]);

File: e2e/wdio/core/tests/feed-input.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+        }
+    });
+
+    it('should have example with avatar image assigned', () => {
+        const inputAvatarLength = getElementArrayLength(feedInputAvatar);
+        for (let i = 0; i < inputAvatarLength; i++) {
+            waitForPresent(feedInputAvatar, i);
+            expect(getAttributeByName(feedInputAvatar, 'image')).not.toBe('');
+            expect(getAttributeByName(feedInputAvatar, 'image')).not.toBeNull();
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/feed-input.e2e-spec.ts:71]: only checking index 0 on assertions, need to add 3rd variable 'i'
(feedInputAvatar, 'image', i)

File: e2e/wdio/core/tests/feed-input.e2e-spec.ts [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+        expect(doesItExist(feedInputNoAvatar)).toBe(true);
+    });
+
+    it('should have Send button assigned and to be disabled if no value set in the input', () => {
+        const inputButtonLength = getElementArrayLength(feedInputButton);
+        for (let i = 0; inputButtonLength > i; i++) {
+            waitForPresent(feedInputButton, i);
+            scrollIntoView(feedInputButton, i);
+            expect(elementDisplayed(feedInputButton, i)).toBe(true);
+            expect(getAttributeByName(feedInputButton, 'aria-disabled', i)).toBe('true');
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/feed-input.e2e-spec.ts:91]: same as above

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

**Review by Seamoo13 — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Review by AntonOlkhovskyi — CHANGES_REQUESTED** [review]

**Review by AntonOlkhovskyi — APPROVED** [review]

**Pull request merged**